api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -20,5 +20,18 @@
     <include file="liquibase-update-to-2.1.xml"/>
     <include file="liquibase-update-to-2.2.xml"/>
     <include file="liquibase-update-to-2.3.xml"/>
-
+<changeSet id="Trunk-5411:Add doseLimitUnit field to Drug" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="drug" columnName="dose_Limit_Unit"/>				
+			</not>
+		</preConditions>
+		<comment>Adding column doseLimitUnit in table drug</comment>
+        <addColumn tableName="drug">
+            <column name="dose_Limit_Unit" type="int">
+            	<constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+    
```

Comment:
@dkayiwa are these changes enough to answer your column names naming conventions?

Reply:
Just list a few column names here together with yours.

Reply:
<column name="cause_of_death_non_coded" type="varchar(255)" />
<column name="speciality_id" type="int" />

  <column name="dose_Limit_Unit" type="int">

Reply:
ooh thanks @dkayiwa,,  i have seen the difference here because of using capital letters

Reply:
@dkayiwa i have updated the changeSet column name